float64 x
float64 y
float64 z
---
bool success
geometry_msgs/Point placeholder
